diff -up transmission-3.00/qt/qtr.pro.omv~ transmission-3.00/qt/qtr.pro
--- transmission-3.00/qt/qtr.pro.omv~	2022-10-26 17:34:19.395303654 +0200
+++ transmission-3.00/qt/qtr.pro	2022-10-26 17:34:23.558334399 +0200
@@ -35,7 +35,7 @@ LIBS += $${DHT_LIBS}
 LIBS += $${LIBB64_LIBS}
 LIBS += $${LIBUPNP_LIBS}
 LIBS += $${LIBNATPMP_LIBS}
-unix: LIBS += -L$${EVENT_TOP}/lib -lz -lrt
+unix: LIBS += -lz -lrt
 win32:LIBS += -levent-2.0 -lws2_32 -lintl
 win32:LIBS += -lidn -liconv -lwldap32 -liphlpapi
 
